src/main/java/io/takari/jdkget/osx/OsxJDKExtractor.java

```diff
@@ -33,10 +33,16 @@ public boolean extractJdk(JdkVersion jdkVersion, File jdkDmg, File outputDirecto
     output.info("Extracting osx dmg image into " + outputDirectory);
     
     // DMG <-- XAR <-- GZ <-- CPIO
-    UnHFS.unhfs(jdkDmg, inProcessDirectory);
+    try {
```

Comment:
I think it's better to let the exception be thrown and instead catch+wrap it in JdkGetter in addition to checking for the boolean result